src/test/unit/lib/model/data/requestToPayRequest.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
@ksatya77 are you sure this is right? It looks like an outbound transfer request rather than an outbound request to pay request.

Reply:
Updated the request now...